consensus/bor/bor.go

```diff
@@ -892,13 +895,16 @@ func (c *Bor) GetCurrentSpan(snapshotNumber uint64) (*Span, error) {
 }
 
 // GetCurrentValidators get current validators
-func (c *Bor) GetCurrentValidators(snapshotNumber uint64, blockNumber uint64) ([]*Validator, error) {
+func (c *Bor) GetCurrentValidators(headerHash common.Hash, blockNumber uint64) ([]*Validator, error) {
```

Comment:
This function caller should rather pass `context` as first argument, instead of 

```go
ctx, cancel := context.WithCancel(context.Background())
```

But we can do it later too.